cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -1916,9 +1920,10 @@ if (roofit_multiprocess)
       unset(cppzmq_${suffix} CACHE)
     endforeach()
     if(fail-on-missing)
-      find_package(cppzmq REQUIRED)
+      # we only support MODULE mode fow now to handle version checking ourselves
+      find_package(cppzmq MODULE REQUIRED)
```

Comment:
Actually, although my comment in `Findcppzmq` is still valid, I see that `cppzmq` actually exports CMake config modules already, so it's much better to just use that instead.